### PR TITLE
[GTK] Switch to newer NSPasteboardType values.

### PR DESCRIPTION
**Pull request by bratsche**
Waiting on https://github.com/mono/mono/pull/18080 to get a build to test with.

### CODE REVIEW

**Comment by bratsche** [comment]
btw this doesn't yet fix GTK's use of `NSFilenamesPboardType` because I didn't see an obvious replacement for it.

File: packages/gtk+.py [hunk 1]
```diff
@@ -226,7 +226,11 @@ def __init__(self):
                 'patches/gtk/nsview-embedding-skip-hidden-subviews.patch',
                 'patches/gtk/0001-gtk-combo-box-native-menu-hook.patch',
 
-                'patches/gtk/gtkviewport-autoscroll.patch'
+                'patches/gtk/gtkviewport-autoscroll.patch',
+
+                # https://devdiv.visualstudio.com/DevDiv/_workitems/edit/993471
+                'patches/gtk/gtk-pboard-types.patch',
+		'packages/patches/gtk/define-NSPasteboardTypeURL.patch'
```
Comment (akoeplinger) [inline packages/gtk+.py]: @bratsche looks like you forgot to push this file. please also fix the indentation to use spaces.

**Comment by akoeplinger** [comment]
@monojenkins backport 2019-12
@monojenkins backport 2019-10

**Comment by akoeplinger** [comment]
@monojenkins backport 2019-08

**Pull request merged**